tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -130,6 +130,35 @@ public async Task InstrumentsDotNetTest()
 
             Task<string> RunDotnet(string arguments) => RunDotnetCommand(workingDir, agent, arguments);
         }
+
+        [SkippableFact]
```

Comment:
This would mean we only run these tests for .NET 8 - given we have different implementations for .NET FX, I wonder if we should run these in all TFMs, WDYT?
```suggestion
#endif

        [SkippableFact]
```

Reply:
ah yes, right, I figured the argument "it's always the same" worked, but it doesn't indeed.